README.md

```
@@ -1,143 +1,535 @@
+   # Get local credentials
+   supabase status
+
+   # Update .env with local credentials
+   NEXT_PUBLIC_SUPABASE_URL="http://localhost:54321"
+   NEXT_PUBLIC_SUPABASE_ANON_KEY="your-local-anon-key"
+   SUPABASE_SERVICE_ROLE_KEY="your-local-service-role-key"
+   POSTGRES_URL="postgresql://postgres:postgres@localhost:54322/postgres"
+   ```
+
+3. **Initialize Database**
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)